src/services/expensesService.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../dataBase');
+
+const Expenses = sequelize.define('Expenses', {
+  id: {
+    type: DataTypes.STRING,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+}, {
+  tableName: 'expenses',
+  updatedAt: false,
+  createdAt: false,
+});
```

Comment:
maybe move it to a different file as well

Reply:
> maybe move it to a different file as well

Good point 👍 
Also if you want to use UUID as `id` the sequelize has corresponding type and you can set value by default so ORM will generate ids for you, and you don't need any other libraries https://sequelize.org/docs/v7/models/data-types/#uuids